router.py

```diff
@@ -0,0 +1,85 @@
+class Module:
+    __idcounter = 0
+    allmodules = []
```

Comment:
Wieso ist idcounter private, aber allmodules public?
Dies gilt für alle anderen Klassen ebenso.

```suggestion
    __allmodules = []
```